dataland-frontend/tests/e2e/specs/data-request/DataRequestProcess.ts

```diff
@@ -209,9 +209,9 @@ describe("As a user I expect a data request page where I can download an excel t
         });
 
         cy.get('button[name="back_to_home_button"]').click();
-        cy.wait(3000);
+        cy.url().should("eq", getBaseUrl() + "/companies");
         cy.get("img.d-triangle-down").click().get("a#profile-picture-dropdown-data-request-button").click();
-        cy.wait(2000);
```

Comment:
Those wait statements are not needed anymore? Were they needed in the first place?

Reply:
I thought they were the only solution to guarantee that the old page is no longer there because there is no intercept for frontend url changes. However, this is of course possible by checking the url.

Reply:
Ok